frontend/app/test/page.tsx

```diff
@@ -27,6 +27,20 @@ import { set } from 'mongoose';
 
 const App = () => {
 
+  const getMeeting = async () => {
+    const meetingId = "2024-04-29T21:01:39.214Z"; // Hardcoded meeting ID, replace with a valid ID for testing
+    try {
+      const response = await fetch(`/api/retrieve/meeting`);
```

Comment:
You define the meetingId constant but don't actually use it in this API call. You should replace this line here with ```const response = await fetch(`/api/retrieve/meeting/${meetingId}`);``` to make sure you're actually using the meetingId in the path. I think this is why your getting an array of multiple meetings as the result as opposed to just the single desired meeting.